app/views/configurations/_configuration_form.html.erb

```diff
@@ -15,12 +15,27 @@
       <% unless configuration.persisted? %>
         <div class="d-flex flex-column align-items-center col-12 px-5 mb-4">
           <h4>Configuration pour : *</h4>
-          <%= f.association :motif_category %>
+          <div class="select">
+            <select
+              data-controller="motif-category-select" 
+              name="configuration[motif_category_id]"
+              data-action="change->motif-category-select#selectChanged"
+              id="motif_category_id"
+            >
+              <option value=""> - </option>
```

Comment:
Il y a une raison spécifique à ne pas utiliser les tag helpers de rails ? On le force pas ailleurs donc il n'y a pas de raison qu'on le force ici, c'est juste par curiosité :)

Reply:
Alors oui c'est parce qu'il me semble que je ne pouvais pas ajouter mon data attribute sur les options `data-participation_optional` avec le helper de rails. Il permet uniquement de faire ce genre de truc sur le select parent.

Reply:
Ok got it merci pour l'explication 🙌 !